src/org/datafoodconsortium/connector/codegen/ruby/static/spec/parse_with_skos_concept_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+describe "parse with skos concept" do
+  let(:connector) { DataFoodConsortium::Connector::Connector.instance }
```

Comment:
I had a look at this out curiosity, the connector instance define in spec helper is different that the once you get when calling `DataFoodConsortium::Connector::Connector.instance` in the spec. 
Anyway I did that originally because I worked with the Minitest test, and there some inefficiencies where the parser would run for every test file and load everything when I only needed the Product Types. I see that's not an issue here because the connector instance gets memoized.